test/jdk/sun/net/www/http/KeepAliveStream/KeepAliveStreamCloseWithWrongContentLength.java

```diff
@@ -23,91 +23,162 @@
 
 /*
  * @test
- * @bug 4533243
- * @summary Closing a keep alive stream gives NullPointerException
+ * @bug 4533243 8263364
+ * @summary Closing a keep alive stream should not give NullPointerException and should accept a connection from  a
```

Comment:
The summary should preferably tell what functionality/API/behavior the test is testing.
Sorry I missed that in the original fix to the main line. 
Since this is a backport I agree to keep the `@summary` in sync with what is in the mainline.